pbdf/Issues/ideal/description.xml

```diff
@@ -10,6 +10,7 @@
 	<SchemeManager>pbdf</SchemeManager>
 	<IssuerID>pbdf</IssuerID>
 	<CredentialID>ideal</CredentialID>
+	<DeprecatedSince>1983740400</DeprecatedSince>
```

Comment:
This seems to be in the year 2032 😬

Reply:
sharp! reminds me to not do things in a hurry.